docs/GDBSERVER.md

```diff
@@ -10,7 +10,6 @@ from qiling import *
 
 def test_gdb(path, rootfs):
     ql = Qiling(path, rootfs, output="off")
-    # ql.gdb = None  # disable gdbserver
```

Comment:
dont delete this line, you can replace it with: 

`# ql.gdb = True  # enable gdbserver at default port`